Source/NSButtonCell.m

```diff
@@ -1703,42 +1705,35 @@ - (void) encodeWithCoder: (NSCoder*)aCoder
       [aCoder encodeValueOfObjCType: @encode(BOOL)
               at: &tmp];
 
-      if([NSButtonCell version] <= 2)
-	{
-	  unsigned int ke = _keyEquivalentModifierMask << 16;
-	  [aCoder encodeValueOfObjCType: @encode(unsigned int)
-		  at: &ke];
-	}
-      else
-	{
-	  [aCoder encodeValueOfObjCType: @encode(unsigned int)
-		  at: &_keyEquivalentModifierMask];
-	}
+      tmp2 = _keyEquivalentModifierMask;
+      [aCoder encodeValueOfObjCType: @encode(uint32_t)
+                                 at: &tmp2];
 
+      tmp2 = _highlightsByMask;
+      [aCoder encodeValueOfObjCType: @encode(uint32_t)
+              at: &tmp2];
+      tmp2 = _showAltStateMask;
       [aCoder encodeValueOfObjCType: @encode(unsigned int)
-              at: &_highlightsByMask];
-      [aCoder encodeValueOfObjCType: @encode(unsigned int)
-              at: &_showAltStateMask];
-
-      if([NSButtonCell version] >= 2)
-	{
```

Comment:
The same story here as the previous comment.  I believe removing the versioning code in encodeWithCoder: will make things much easier to understand.